backend/Techfesia2019/Techfesia2019/settings.py

```diff
@@ -30,15 +30,25 @@
 
 # Application definition
 
+
 INSTALLED_APPS = [
+    'registration.apps.RegistrationConfig',
```

Comment:
there is no need of writing `registration.apps.RegistrationConfig`
You can just write `registration`